utils/db/repos.ts

```diff
@@ -170,3 +170,50 @@ export const getRepoConfigByUserAndRepo = async (provider: string, repoName: str
 	}
 	return userRows[0].config;
 }
+
+export const insertRepoConfig = async (userId: string, repoIds: number[]) => {
+	const query = `
+		INSERT INTO repo_config (repo_id, user_id, auto_assign, comment_setting)
+		SELECT id, $1, true, true
+		FROM unnest($2::INT[]) AS t(id)
+		ON CONFLICT DO NOTHING
+		`;
+	const params = [userId, repoIds];
+
+	const queryIsSuccessful = await conn.query(query, params)
+	.then((dbResponse) => {
+		if (dbResponse.rowCount == 0) {
+			return false;
+		}
+		return true;
+	})
+	.catch((err: Error) => {
+		console.error(`[db/insertRepoConfigOnSetup] Could not insert repo config for the repos: ${repoIds}`, { pg_query: query }, err);
+		return false;
+	})
+	return queryIsSuccessful;
+}
+  
+export const removeRepoconfigForInstallId = async (install_id: string, repo_names: string[], provider: string, user_id: string) => {
+    const deleteRepoConfigQuery = `
+        DELETE FROM repo_config
+        WHERE repo_id IN (
+        SELECT id
+        FROM repos
+        WHERE install_ids && ARRAY[$1]
```

Comment:
what does this line do? `install_ids && ARRAY[$1]`

Reply:
it basically checks if there exist an overlap between the install_ids array and the array we created in the query.